src/app.js

```
@@ -43,18 +44,50 @@
+    const header = document.querySelector('header');
+    let countdownEl = document.getElementById('activity-countdown');
+    if (!countdownEl) {
+      countdownEl = document.createElement('div');
+      countdownEl.id = 'activity-countdown';
+      countdownEl.style.margin = '10px 0';
+      countdownEl.style.fontWeight = 'bold';
+      header.appendChild(countdownEl);
+    }
+    const updateCountdown = () => {
+      const now = new Date();
+      let diff = Math.max(0, this.activityEndTime - now);
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'diff' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)